src/components/SessionApiLogout.tsx

```diff
@@ -0,0 +1,18 @@
+require('dotenv').config();
+
+import * as React from 'react';
+
+export const SessionApiLogout = async () => {
+  const logoutUrl: string = process.env.REACT_APP_API_URL + '/sign_out';
+
+  await fetch(logoutUrl, { method: 'DELETE' }).then((response) => {
+    if (process.env.NODE_ENV !== 'production') {
+      console.log('success');
+      console.log('reaponse(sessionApi.login)');
+      console.log(response);
+    }
+    const loginStatus = 'false';
+
+    return loginStatus;
```

Comment:
Is there any reason why SessionApiLogout always returns false

please make error handler  

Is there any reason why  loginStatus is "false", n ot false